### ISSUE TITLE
Dataset ID Doesn't Work for datasetID

### ISSUE THREAD

**Issue by DarioS** [SydneyBioX/BenchHub#11]
Creating a new Trio doesn't work if a dataset ID is specified to `datasetID`!

```r
> datasets[1:3, 1:4]
# A tibble: 3 × 4
  datasetID name     source   sourceID         
  <chr>     <chr>    <chr>    <chr>            
1 0000      GE1_data figshare 26142922/47361037
2 0001      GE2_data figshare 26142922/47361052
3 0002      GE3_data figshare 26142922/47361049

> Trio$new("0001")
✔ Reading from Curated Trio Datasets.
✔ Range ''Datasets''.
Error in `private$parseIDString()`:
! Specified dataset ("0001") is not avaiable.
```

Only source ID works, despite

> `datasetID`: A string specifying a dataset, either a name from curated-trio-data or a format string of the form `source`:`source_id`.

**Comment by nick-robo** [comment]
It's a little confusing, but the datasetID in Trio is the name in curated datasets. So to get dataset “0001”, you would give trio “GE1_data”. 

Should I make the documentation clearer on this issue?

**Comment by DarioS** [comment]
Yes, names (e.g. gene names) are conventionally not unique, but IDs conventionally are. If the Trio class contructor's parameter name is `datasetID` I imagine that a lot of people would pick the datasetID column from the spreadsheet.
